lua/entities/gmod_wire_expression2/core/string.lua

```diff
@@ -39,19 +39,58 @@ end)
 registerOperator("is", "s", "n", function(self, args)
 	local op1 = args[2]
 	local rv1 = op1[1](self, op1)
-	if rv1 ~= "" then return 1 else return 0 end
+
+	return rv1 ~= "" and 1 or 0
 end)
 
 registerOperator("eq", "ss", "n", function(self, args)
 	local op1, op2 = args[2], args[3]
 	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
-	if rv1 == rv2 then return 1 else return 0 end
+
+	return rv1 == rv2 and 1 or 0
 end)
 
 registerOperator("neq", "ss", "n", function(self, args)
 	local op1, op2 = args[2], args[3]
 	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
-	if rv1 != rv2 then return 1 else return 0 end
+
+	return rv1 ~= rv2 and 1 or 0
+end)
+
+registerOperator("geq", "ss", "n", function(self, args)
+	local op1, op2 = args[2], args[3]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+
+	self.prf = self.prf + math.min(#rv1, #rv2) / 10
```

Comment:
@Divran @syranide Do you think there will be an issue with non-integer ops?
Will they display properly?

Reply:
We already [use non-integer op counts](https://github.com/wiremod/wire/blob/fa855d63e452ece3b57dd652196afa4e0280d925/lua/entities/gmod_wire_expression2/core/array.lua#L524-L527) for various array and table functions.

Reply:
cool
move along, nothing to see :)